languages/rust/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -20,11 +27,4 @@ The following combinations are possible:
 - 64 bits: `u64`, `i64`, `f64`
 - 128 bits: `u128`, `i128`
 
-(note that there are only 32-bits and 64-bits variants for floating-point numbers)
-
-## Converting between number types
-
-Rust doesn't do any implicit type conversion.
-This means that if you need to turn one numeric type into another, you have to do so explicitly.
-When converting from a larger type to a smaller one (for instance `u64` to `u32`) you could lose data.
-Converting from a floating point to an integer **will** lose everything behind the decimal point, effectively rounding down.
```

Comment:
This is useful information; please don't remove it. It could use wording improvements, but that could be part of a distinct PR.